src/definitions/modules/modal.js

```diff
@@ -865,6 +898,18 @@
                         });
                         module.cache.topOffset = -(module.cache.height / 2);
                     }
+                    if (getScrollingMargins) {
+                        if (module.can.useFlex()) {
+                            $module.addClass(className.scrolling);
+                            scrollingTop = parseInt($module.css('top').replace(/[^\d.]/g, ''), 10) || 0;
```

Comment:
Is the `top` css set by us - if yes, we should know the format and something like `.replace(/px$/, '')` would be much more cleaner.